CleverTap/Runtime/Native/UnityNativeWrapper/UnityNativePreferenceManager.cs

```diff
@@ -54,16 +55,34 @@ internal void SetInt(string key, int value)
 
         internal long GetLong(string key, long defaultValue)
         {
-            // Use string to get long values in PlayerPrefs
-            return long.Parse(PlayerPrefs.GetString(GetStorageKey(key), defaultValue.ToString()));
+            return long.Parse(PlayerPrefs.GetString(GetStorageKey(key), defaultValue.ToString(CultureInfo.InvariantCulture)));
         }
 
-        internal void SetLong(string key, long value)
+        internal void SetLong(string key, string longValue)
```

Comment:
SetLong should accept `long` longValue, not string.

Reply:
Done